ui.apps/src/main/content/jcr_root/apps/core/fd/components/aemform/v1/aemform/clientlibs/runtime/formapp/js/formapp.js

```diff
@@ -87,8 +87,8 @@
                 }, aemFormIframeSelector);
             }
             if (aemFormConfig.height != "auto") {
-                $(aemFormIframeSelector).css("height", aemFormConfig.height);
-            }
+                document.querySelector(aemFormIframeSelector).style.height = aemFormConfig.height;
+	    }
             var submitConfig = {},
                 renderConfig = {},
```

Comment:
Add a test for this usecase.

Reply:
added a test case for this functionality